lib/deck.rb

```diff
@@ -0,0 +1,11 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards = [])
```

Comment:
You don't need to pass it as a default value. If you look at the interaction pattern, cards is not empty.